src/components/Review.jsx

```diff
@@ -49,6 +61,56 @@ const Review = () => {
     });
   };
 
+  const handleCommentSubmit = event => {
+    event.preventDefault();
+    const commentBody = event.target['0'].value;
+    postComment(user, review.review_id, commentBody)
+      .then(newComment => {
+        setComments(currComments => {
+          const newComments = [...currComments];
+          newComments.unshift(newComment);
+          return newComments;
+        });
+        setIsSubmitted(true);
+      })
+      .catch(err => {
+        if (err.response.data.msg === 'User does not exist') {
+          setUserExists(false);
+        }
+      });
+  };
+
+  let commentContainer;
+
+  isSubmitted
+    ? (commentContainer = <p>comment added!</p>)
+    : (commentContainer = (
+        <div>
+          <h3>Add new comment</h3>
+          <form onSubmit={handleCommentSubmit} id="new-comment">
```

Comment:
Forms should be controlled  in this case (both value and handleChange set for inputs) :)